blocks/header/header.js

```diff
@@ -93,21 +105,35 @@ function toggleNavSubSection(event) {
   }
 }
 
+function toggleBreadCrumb() {
+  document.querySelector('.menu-option-sublist.l3-nav-menu-options').classList.toggle('active');
```

Comment:
Didn't opt to add a check on the current state or the trigger to control explicitly whether to set the relevant classes or not, just relying on toggle since I am fairly certain this is the only place these elements  will be controlled.